js/w3c/style.js

```diff
@@ -10,8 +10,6 @@ define(
     function(utils, pubsubhub) {
       function attachFixupScript(doc, version){
         var script = doc.createElement("script");
-        script.async = true;
-        script.defer = true;
```

Comment:
So, my reading is that because this script is pre-loaded, we don't need to defer it. And as it's already at the end of the document (and added after processing), async won't matter.